scenario/utils.ts

```diff
@@ -254,3 +267,199 @@ export async function fetchLogs(
     return contract.queryFilter(filter, fromBlock, toBlock);
   }
 }
+
+export async function executeOpenProposal(
+  dm: DeploymentManager,
+  { id, startBlock, endBlock }: OpenProposal
+) {
+  const governor = await dm.contract('governor');
+
+  if (!governor) {
+    throw new Error("cannot execute proposal without governor");
+  }
+
+  const blockNow = await dm.hre.ethers.provider.getBlockNumber();
+  const blocksUntilStart = startBlock - blockNow;
+  const blocksUntilEnd = endBlock - Math.max(startBlock, blockNow);
+
+  if (blocksUntilStart > 0) {
+    await mineBlocks(dm, blocksUntilStart);
+  }
+
+  if (blocksUntilEnd > 0) {
+    for (const whale of COMP_WHALES) {
```

Comment:
`COMP_WHALES` would be chain dependent, so let's leave a XXX to properly handle this

Reply:
good call; added